R/declare_estimator.R

```diff
@@ -369,10 +360,18 @@ hasS3Method <- function(f, obj) {
 # called by model_handler, resets columns names !!!
 fit2tidy <- function(fit, term = FALSE) {
   
+  # browser()
   if (hasS3Method("tidy", fit)) {
     tidy_df <- tidy(fit, conf.int = TRUE)
   } else {
-    tidy_df <- tidy_default(fit, conf.int = TRUE)  
+    tidy_df <- try(tidy_default(fit, conf.int = TRUE), silent = TRUE)
```

Comment:
This seems fine, but will swallow the original error message from tidy_default. You may want to also print it out to make future issues easier to triage.